docs/guides/run-jobs-batch.mdx

```diff
@@ -69,7 +69,7 @@ Default values:
 | --- | --- | --- |
 | Both Open and Premium Plan | 1 sec | 10 min |
 
```

Comment:
I can't comment on previous lines, but let's add some detail to the first line to help people who are searching with other terms: "You can define the batch's maximum time to live (TTL) with the `max_time` parameter."  Maybe something like this, to match what was added in the session section:

"The maximum TTL determines how long each batch job can run before it expires.  You can set this value with the `max_time` parameter."